common-android/build.gradle

```diff
@@ -39,9 +50,6 @@ android {
 
 dependencies {
     api project(":common")
-    implementation 'androidx.core:core-ktx:1.7.0'
-    implementation 'androidx.appcompat:appcompat:1.4.1'
-    implementation 'com.google.android.material:material:1.5.0'
```

Comment:
Not sure what these were used for so I removed them, lmk if I should add them back.

Reply:
I guess these were copied earlier from android, we can remove this if this won't hurt build. Do you still want to have an github for this or manual for now?

Reply:
I'll add an action